packages/wordlist/src/storage/remote-db-adapter.js

```diff
@@ -172,6 +171,9 @@ export default class RemoteDBAdapter {
           this.errors.push(error)
         }
       }
+      if (error.message === 'Request failed with status code 401') {
+        return []
+      }
```

Comment:
why the change in behavior here?

Reply:
oh, I added it after problems with testUser - without this it was failed with unproper result
I belive that it is better to simply get empty array and of course show all errors as before including this 401 error

Reply:
ok, thanks!